cli/tests/integrations/test_marathon_pod.py

```diff
@@ -107,21 +109,24 @@ def _assert_pod_list_json_subset(expected_json, actual_json):
     for expected_pod in expected_json:
         pod_id = expected_pod['id']
         actual_pod = actual_pods_by_id[pod_id]
+        _assert_pod_json_subset(expected_pod, actual_pod)
 
-        expected_containers = expected_pod['containers']
-        actual_containers = actual_pod['containers']
-        actual_containers_by_name = {c['name']: c for c in actual_containers}
+    assert len(actual_json) == len(expected_json)
 
-        for expected_container in expected_containers:
-            container_name = expected_container['name']
-            actual_container = actual_containers_by_name[container_name]
 
-            for k, v in expected_container['resources'].items():
-                assert actual_container['resources'][k] == v
+def _assert_pod_json_subset(expected_pod, actual_pod):
```

Comment:
lets call this `_assert_pod_json` and add a pydoc explaining thats its just checking specific fields.